rainier-benchmark/benchmarks.txt

```diff
@@ -122,9 +122,9 @@ run                  Geometric                 100                            1.
 run                  Geometric                 1000                           1.000
 run                  Geometric                 10000                          1.000
 run                  Geometric                 100000                         1.000
-run                  GeometricZeroInflated     100                            1.000
-run                  GeometricZeroInflated     1000                        1000.000
-run                  GeometricZeroInflated     10000                      10000.000
+run                  GeometricZeroInflated     100                            2.000
+run                  GeometricZeroInflated     1000                         300.000
+run                  GeometricZeroInflated     10000                       3000.000
```

Comment:
...and these are faster??

Reply:
overall I think this all reads to me as "no change" given the still-imperfect stability of the benchmark numbers. which is good.

Reply:
Yeah, that makes sense.